test/bootstrap.test.js

```diff
@@ -1,52 +1,66 @@
 'use strict';
+/* eslint-disable no-console */
 const sails = require('sails');
+let mongoServerInstance;
 
 before(function(done) {
-
-  // Increase the Mocha timeout so that Sails has enough time to lift.
-  this.timeout(10000);
   require('babel-register')();
 
-  sails.lift({
-    environment: 'test',
-    port: 1234,
-    connections: {
-      default: 'testDB',
-      testDB: {
-        adapter: 'sails-mongo',
-        host: 'localhost',
-        port: 27017,
-        user: '',
-        password: '',
-        database: 'porybox-test'
-      }
-    },
-    models: {
+  const MongoInMemory = require('mongo-in-memory');
+  const port = 5544;
+  mongoServerInstance = new MongoInMemory(port);
+
+  mongoServerInstance.start((error) => {
+    if (error) {
+      throw error;
+    }
+    const mongouri = mongoServerInstance.getMongouri('test');
+
+    sails.lift({
+      environment: 'test',
+      port: 1234,
+      connections: {
+        default: 'testDB',
+        testDB: {
+          adapter: 'sails-mongo',
+          url: mongouri
+        }
+      },
+      models: {
+        connection: 'testDB',
+        migrate: 'drop'
+      },
+      session: {
+        url: mongouri
+      },
       connection: 'testDB',
-      migrate: 'drop'
-    },
-    connection: 'testDB',
-    paths: { public: 'client', views: 'client' },
-    email: {
-      transport: {
-        host: 'localhost',
-        port: 1235,
-        ignoreTLS: true
+      paths: { public: 'client', views: 'client' },
+      email: {
+        transport: {
+          host: 'localhost',
+          port: 1235,
+          ignoreTLS: true
+        },
+        sender: 'Porybox <no-reply@example.org>'
       },
-      sender: 'Porybox <no-reply@example.org>'
-    },
-    log: {level: 'debug'},
-    hooks: {
-      grunt: false
-    }
-  }, function(err) {
-    if (err) return done(err);
-    // here you can load fixtures, etc.
-    done(err, sails);
+      log: {level: 'debug'},
+      hooks: {
+        grunt: false
+      }
+    }, function(err) {
+      if (err) return done(err);
+      // here you can load fixtures, etc.
+      done(err, sails);
+    });
+
   });
 });
 
 after(function(done) {
-  // here you can clear fixtures, etc.
+  mongoServerInstance.stop((error) => {
+    if (error) {
+      console.error(error);
```

Comment:
nitpick: can `no-console` be disabled with an inline comment here rather than the entire file?

```js
console.error(error); // eslint-disable-line no-console
```